core/sr-api-macros/benches/bench.rs

```diff
@@ -53,13 +53,13 @@ fn sr_api_benchmark(c: &mut Criterion) {
 	});
 
 	c.bench_function("calling function by function pointer in wasm", |b| {
-		let client = test_client::new_with_execution_strategy(ExecutionStrategy::AlwaysWasm);
+		let client = test_client::new();
 		let block_id = BlockId::Number(client.info().chain.best_number);
 		b.iter(|| client.runtime_api().benchmark_indirect_call(&block_id).unwrap())
 	});
 
 	c.bench_function("calling function in wasm", |b| {
-		let client = test_client::new_with_execution_strategy(ExecutionStrategy::AlwaysWasm);
```

Comment:
Is this correct?  CI didn’t pass without it.

Reply:
no